students/Vincent_Aquila/session02/printgrid.py

```diff
@@ -0,0 +1,51 @@
+"""
+Python210A Vincent Aquila  Fall2018
+using Python 3.7.0
+assignment objective: grid
+"""
+
+#Part 1
+#Function for top, middle, and bottom of grid
+def grid_one():
+	print('+', '-', '-', '-', '-', '+', '-', '-', '-', '-', '+')
```

Comment:
if you want to print all this will spaces, probablh beter to build up the string, and make it one print.

things like `"- " * 5`  can be helpful.

-CHB